gitlab/__init__.py

```diff
@@ -777,7 +777,14 @@ class GitlabList(object):
 
     def __init__(self, gl, url, query_data, get_next=True, **kwargs):
         self._gl = gl
-        self._query(url, query_data, **kwargs)
+
+        # Preserve kwargs for subsequent queries
+        if kwargs is None:
+            self._kwargs = {}
+        else:
+            self._kwargs = kwargs.copy()
```

Comment:
`kwargs` can't be `None`:

```python
>>> def foo(**kwargs):
...   print(kwargs)
... 
>>> foo()
{}
>>> foo(**None)
Traceback (most recent call last):
  File "<input>", line 1, in <module>
    foo(**None)
TypeError: foo() argument after ** must be a mapping, not NoneType
```

So this  can just be:

```suggestion
        self._kwargs = kwargs.copy()
```